app/views/help/modifiying_requests.md

```diff
@@ -1,28 +1,27 @@
-<% title "What if there's something wrong with the request?" %>
+<% title "What if I need to modify a request?" %>
 
-A request may need to be modified after it is submitted. These situations include:
-
-* There was a typo.
-* A different RWA number needs to be used. (BA80)
-* The original request had a Not-To-Exceed amount, which needs to be changed to Exact once it's complete.
+You may need to modify a request after it is submitted for a number of reasons:
+  - To correct a typo or other error
+  - To update a Not-To-Exceed amount to a final purchase price
+  - To change approval information or provide a new RWA number
 
 ## Approver
 
-If you are one of the approvers and notice an issue:
+Approvers cannot modify requests, but they can ask the requester to do so. If you are one of the approvers and notice an issue:
```

Comment:
How to set up the behavior is up to you. Trying a change in language that opens this up a bit, though.